### ISSUE TITLE
Not working on MIUI14 China version xiaomi13

### ISSUE THREAD

**Issue by nig5ht28** [AndroPlus-org/magisk-module-androidauto#1]
After I plugged the phone into the car, there was no AA prompt  on the screen, the phone flashed a notification (AA connected to the phone)and then disappeared.

**Comment by Noado** [comment]
What version did you used? With "android 11" or without version?

**Comment by Noado** [comment]
Tested v1 at Xiaomi 12Su and Emulator. I'm not quite sure... but as i remember, there was a notification about accepting connection. Could it be that energy-saving and so on features block needed content now? It did not work before for me, just bought the device.

Also, when emulator is starting i can see some black window at phone screen that hides. Old emulator says nothing exept "waiting for device" but phone makes a notification "connected to auto".. well, it's not. Looks like we miss something. Maybe we should not update the app from play store?

Oh, found these
> Activity Launcher (or something) to open Android Auto's ".companion.AndroidAutoActivity"
But still no luck

**Comment by guoweiy** [comment]
I encountered the same issue. 

Device: Xiaomi 13
MIUI Version: 14.0.13.0 TMCCNXM (The latest stable CN ROM)
Magisk Version: 25.2
Using the v1 release package: magisk-module-androidauto-android13.zip

I tried several ways to use it, but nothing works:

1. After installing the magisk module, directly plug the phone to the car. A window pops up, prompting to update the AA app from play store. 

2. I updated the AA app from play store, then reconnected the USB cable. This time a red window pops up, saying communication error 25 and Nearby Devices Permission not granted.

3. Then I opened the .companion.AndroidAutoActivity using Activity Launcher. There are some prompts saying AA needs permission. I tapped it, and AA started to ask for several permissions, including the Nearby Devices Permission. I allowed all of them, of course.

4. After granting all the permissions, I reconnect the cable. A black window quickly jumps up and disappears, and in the notification bar it says AA is connected to the car (need to enable notification for AA). But in the AA app it still says no car is connected, and the projection doesn't work either.

BTW I also tried other ways, like using the Systemizer, but it has the similar outcome.
Any ideas on this? Thanks!

**Comment by darrenlimch** [comment]
> I encountered the same issue.
> 
> Device: Xiaomi 13 MIUI Version: 14.0.13.0 TMCCNXM (The latest stable CN ROM) Magisk Version: 25.2 Using the v1 release package: magisk-module-androidauto-android13.zip
> 
> I tried several ways to use it, but nothing works:
> 
> 1. After installing the magisk module, directly plug the phone to the car. A window pops up, prompting to update the AA app from play store.
> 2. I updated the AA app from play store, then reconnected the USB cable. This time a red window pops up, saying communication error 25 and Nearby Devices Permission not granted.
> 3. Then I opened the .companion.AndroidAutoActivity using Activity Launcher. There are some prompts saying AA needs permission. I tapped it, and AA started to ask for several permissions, including the Nearby Devices Permission. I allowed all of them, of course.
> 4. After granting all the permissions, I reconnect the cable. A black window quickly jumps up and disappears, and in the notification bar it says AA is connected to the car (need to enable notification for AA). But in the AA app it still says no car is connected, and the projection doesn't work either.
> 
> BTW I also tried other ways, like using the Systemizer, but it has the similar outcome. Any ideas on this? Thanks!

Yes, i have mi 13 pro having exactly same problem like you. Flash this module or systemize method also doesnt work.

**Comment by Noado** [comment]
Someone told i need to copy apk file to system folder with apk's. I switched to eu fw and can't check.

**Comment by darrenlimch** [comment]
> Someone told i need to copy apk file to system folder with apk's. I switched to eu fw and can't check.

Dont think will work, cuz when you use the "systemize" method it already copy the aa.apk to the systempriv location. But still not work. Sadly mi13pro too new, eu not launch yet.

**Comment by hykilpikonna** [comment]
For me, I just had to install https://github.com/AndroPlus-org/magisk-module-miui-cn2global. After that, it worked.
